api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java

```diff
@@ -131,15 +131,16 @@ private void validateAndCleanUpConcepts(JdbcConnection connection) {
 			List<ConceptName> namesWithNoLocale = null;
 			
 			//for each locale
-			for (Locale conceptNameLocale : locales) {
```

Comment:
From the code diff i can't see that there was a localeConceptNamesMap.keySet() call. Can you tell why you see that there should "still" be the call?
In my opinion, and that's why i made the pull request, old and new code are equivalent in what they achieve.